src/components/subscriptions/data/utils.js

```diff
@@ -34,24 +34,39 @@ export const getSubscriptionStatus = (subscription) => {
   return ACTIVE;
 };
 
-// Sort plans by statuses, active -> scheduled -> ended.
-export const sortSubscriptionsByStatus = (subscriptions) => subscriptions.slice().sort(
-  (sub1, sub2) => {
-    const orderByStatus = {
-      [ACTIVE]: 0,
-      [SCHEDULED]: 1,
-      [ENDED]: 2,
-    };
-    const sub1Status = getSubscriptionStatus(sub1);
-    const sub2Status = getSubscriptionStatus(sub2);
+/**
+ * Sort subscription plans by:
+ *   - Statuses (active -> scheduled -> ended)
+ *   - Plans within same status, sorted by expiration date (ascending)
+ *   - Plans within same status and expiration date, sorted by title (ascending)
+ *
+ * @param {Array} subscriptions - List of subscription plans.
+ *
+ * @returns Ordered list of subscription plans.
+ */
+export const sortSubscriptionsByStatus = (subscriptions) => {
+  const statusOrder = {
+    [ACTIVE]: 0,
+    [SCHEDULED]: 1,
+    [ENDED]: 2,
+  };
+  return subscriptions.slice().sort(
+    (sub1, sub2) => {
+      const sub1Status = getSubscriptionStatus(sub1);
+      const sub2Status = getSubscriptionStatus(sub2);
+
+      if (statusOrder[sub1Status] !== statusOrder[sub2Status]) {
+        return statusOrder[sub1Status] - statusOrder[sub2Status];
+      }
 
-    if (sub1Status === sub2Status) {
-      return dayjs(sub1.startDate) - dayjs(sub2.startDate);
-    }
+      if (sub1.expirationDate !== sub2.expirationDate) {
```

Comment:
[inform] Uses expiration date of the plan, for parity with Learner Credit Management. Discussed and decided with Jenn Gregory.